tests/simulations/ringtest/create_data.py

```diff
@@ -74,6 +74,7 @@ def make_ringtest_nodes():
         "y": it.count(1),
         "z": it.count(2),
         "morphology": "cell_small",
+        "synapse_class": ['EXC', 'EXC', 'EXC']
```

Comment:
Is `synapse_class` required for the ringtest ?

Reply:
good point, I think not, as I'm not replacing the h5 files; I wonder where that came from.
I will remove it.